external/loaders/loaders/_utils.py

```diff
@@ -81,7 +79,7 @@ def add_wind_rotation_info(res: str, ds: xr.Dataset) -> xr.Dataset:
         res: grid resolution, format as f'c{number cells in tile}'
     """
 
-    rotation = _load_wind_rotation_matrix(res)
+    rotation = _load_wind_rotation_matrix(res).drop("tile")
```

Comment:
Unrelated to this PR, but fixes the bug that appears if you merge datasets that disagree on starting tile index.